designer/app.py

```diff
@@ -135,6 +136,17 @@ class Designer(FloatLayout):
        a :class:`~kivy.properties.BooleanProperty`
     '''
 
+    _quit_on_save = BooleanProperty(False)
```

Comment:
we should avoid creating this new variable. It's a variable to create a kind of restriction, let's keep it clean. See the comments below why it's not necessary